src/test/java/net/sf/jabref/importer/fileformat/BibtexParserTest.java

```diff
@@ -1631,4 +1631,25 @@ public void parseCommentContainingEntries() throws IOException {
         assertEquals(bibtexEntry, entry.getParsedSerialization());
     }
 
+    @Test
+    public void parseCommentContainingEntriesAndAtSymbols() throws IOException {
+        // @formatter:off
+        String bibtexEntry = "@Comment{@article{myarticle,}" + OS.NEWLINE +
+                "@inproceedings{blabla, title={the proceedings of bl@bl@}; }" + OS.NEWLINE +
+                "} " + OS.NEWLINE +
+                "@Article{test," + OS.NEWLINE +
+                "  Author                   = {Foo@Bar}," + OS.NEWLINE +
+                "  Journal                  = {International Journal of Something}," + OS.NEWLINE +
+                "  Note                     = {some note}," + OS.NEWLINE +
+                "  Number                   = {1}" + OS.NEWLINE +
+                "}";
+        // @formatter:on
+
+        ParserResult result = BibtexParser.parse(new StringReader(bibtexEntry));
+        Collection<BibEntry> entries = result.getDatabase().getEntries();
+        BibEntry entry = entries.iterator().next();
+
+        assertEquals(bibtexEntry, entry.getParsedSerialization());
+    }
```

Comment:
Besides checking the parsed serialization, could you please also add another test which checks that an entry containing an at symbol is parsed correctly, i.e. has the correct field values.